osclib/pkglistgen_comments.py

```diff
@@ -271,7 +361,9 @@ def check_staging_accept(self, project, target):
         sections = self.parse_sections(comment['comment'])
         with tempfile.TemporaryDirectory() as tmpdirname:
             checkout_package(self.apiurl, target, '000package-groups', expand_link=True, outdir=tmpdirname)
+            # Now should go to Attribute OSRT:summary-staging in the package or project
```

Comment:
Not sure this is true anymore

Reply:
Yes I will drop this. This was a WIP comment that slipped into the commit.